### DISCUSSION TITLE
Imposter-Syndrome

### DISCUSSION THREAD

**Discussion by Tobinchilongo** [community/community#21092] [Projects and Issues] [answered]
### After the final capstone: 

There is that feeling that kicks in and leaves you wondering whether or not you are worthy or ready to go out there and start job hunting...

How to overcome that 😭

**Comment by radar-blip** [comment, marked as answer]
Just do it!

**Reply by Tobinchilongo** [reply]
Thank you

**Reply by radar-blip** [reply]
No problem. Its not worth it to get all worked up about stuff. You'll just make yourself a nervous wreck, its not worth it. lol

**Comment by nemariia** [comment]
Hey, you are worthy! You can do this! Job seeking is a special skill, and you can get it as any other skill - through practice.

**Reply by Tobinchilongo** [reply]
Thanks a lot for those kind words

**Reply by nemariia** [reply]
You're welcome! :slightly_smiling_face:
